alf/drivers/policy_driver.py

```diff
@@ -113,12 +113,20 @@ def get_initial_time_step(self):
                                                      self.env.batch_size)
         return make_action_time_step(time_step, action)
 
-    def algorithm_step(self, time_step, state):
+    def algorithm_step(self, time_step, state, training=self._training):
```

Comment:
Perhaps we should remove the default value of training to make it a little easier to understand the logic.

Reply:
removed